processor/processor.go

```diff
@@ -265,11 +272,6 @@ func (p *processor) processType(pkg *loader.Package, info *markers.TypeInfo, dep
 		Doc:     info.Doc,
 	}
 
-	if p.shouldIgnoreType(types.Key(typeDef)) {
-		zap.S().Debugw("Skipping excluded type", "type", typeDef.String())
-		return nil
-	}
-
```

Comment:
I think this breaks the `ignoreTypes` setting which seems to be now ignored: https://github.com/elastic/cloud-on-k8s/pull/7608/files#discussion_r1528299014